apps/mull-api/src/schema.gql

```diff
@@ -3,12 +3,17 @@
 # ------------------------------------------------------
 
 input CreateUserInput {
-  age: String!
+  dob: DateTime!
   email: String!
   name: String!
   password: String!
 }
 
+"""
+A date-time string at UTC, such as 2019-12-03T09:54:33Z, compliant with the date-time format.
+"""
+scalar DateTime
+
 type Mutation {
```

Comment:
Probably should be ignored cuz it's generated. Create a gitignore in apps/mull-api so we modularize our gitignore files and add.
`/src/schema.gql`

Reply:
I think it might be a good idea to leave it in version control, to make sure everyone has it and to keep track of changes to our schema, kinda like how package-lock.json is in version control usually.

Reply:
We can add it to gitignore in a separate PR if need be, since these changes are urgent, sorry laurendy